submissions/Border Walls/border-walls.md

```diff
@@ -0,0 +1,11 @@
+
+Now, everyone is familiar with the concept of building walls around your factory, and fortifying them with turrets. People will argue over whether lasers, gun turrets, or flamethrowers are the best, and if you're playing mods like Bob's you might have access to other options such as plasma turrets too. What people often fail to realize, though, is that the shape of the wall makes a huge difference. In my youth, when I first discovered Factorio, I tended to build walls around places that got attacked, and usually around my entire factory as well. I would build them right at the perimeter - after all, defending farther out than was necessary would be a waste of resources and would force me to clear countless nests of biters. But then I discovered something that changed all that: the magic of the choke point. By using natural cliffs, bodies of water, and the fact that pollution only spreads so far, it's possible to create a border wall much further out than I originally realized - and not only is it possible; it's actually advantageous.
+
+Building a border wall farther out than the edge of your factory has numerous advantages. First and foremost, it allows you to choose where you want to deal with bugs. If you clear out to a choke point and then hold that choke, you don't have to worry about attacks from that direction - and if you do this in all directions, you don't have to worry about attacks at all. Pushing out past your pollution boundary means that bugs won't even attack at these defenses: the only thing the turrets are there for is to stop the expansion of bugs into already-cleared territory. By using natural barriers such as cliffs and water to aid your defenses, you can cut off swathes of the map with very little use of resources: I would estimate somewhere between forty and eighty percent of my border wall tends to be naturally-formed, impassible obstacles. With trees and tiles absorbing my pollution instead of biter spawners, I don't even get very frequently attacked - and this is crucial, because it allows me to throw down a couple of hundred rounds of piercing ammo at a chokepoint and leave it alone for several hours.
```

Comment:
```suggestion
Building a border wall farther out than the edge of your factory has numerous advantages. First and foremost, it allows you to choose where you want to deal with biters. If you clear out to a choke point and then hold that choke, you don't have to worry about attacks from that direction - and if you do this in all directions, you don't have to worry about attacks at all. Pushing out past your pollution boundary means that bugs won't even attack at these defenses: the only thing the turrets are there for is to stop the expansion of bugs into already-cleared territory. By using natural barriers such as cliffs and water to aid your defenses, you can cut off swathes of the map with very little use of resources: I would estimate somewhere between forty and eighty percent of my border wall tends to be naturally-formed, impassible obstacles. With trees and tiles absorbing my pollution instead of biter spawners, I don't even get very frequently attacked - and this is crucial, because it allows me to throw down a couple of hundred rounds of piercing ammo at a chokepoint and leave it alone for several hours.
```
Probably should use biters throughout rather than bugs as it could be confusing.